draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -311,19 +311,27 @@ Each DNS request-response pair is matched to one HTTP request-response
 pair. The responses may be processed and transported in any order
 using HTTP's multi-streaming functionality ({{RFC7540}} Section 5}).
 
-The Answer section of a DNS response contains one or more RRsets.
+The Answer section of a DNS response can contain zero or more RRsets.
 (RRsets are defined in {{RFC7719}}.)  According to {{RFC2181}}, each
-resource record in an RRset is supposed to have the Time To Live (TTL)
+resource record in an RRset has Time To Live (TTL)
 freshness information.  Different RRsets in the Answer section can
-have different TTLs, though it is only possible for the HTTP response
+have different TTLs, although it is only possible for the HTTP response
 to have a single freshness lifetime.  The HTTP response freshness lifetime
-({{RFC7234}} Section 4.2) should be coordinated with the Resource
-Record bearing the smallest TTL in the Answer section of the response.
+({{RFC7234}} Section 4.2) should be coordinated with the RRset
+with the smallest TTL in the Answer section of the response.
 Specifically, the HTTP freshness lifetime SHOULD be set to expire at the same time
-any of the DNS Records reach a 0 TTL.  The response freshness lifetime
-MUST NOT be greater than that indicated by the DNS Record with the
+any of the DNS resource records in the Answer section reach a 0 TTL.
+The response freshness lifetime
+MUST NOT be greater than that indicated by the DNS resoruce record with the
 smallest TTL in the response.
 
+If the DNS response has no records in the Answer section, 
+and the DNS response has an SOA record in the Authority section,
+the response freshness lifetime MUST be set to the MINIMUM field from
+that SOA record. Otherwise, the server MUST
+use the "Cache-Control no-cache" directive in the
```

Comment:
I'll find words for "make uncachable" which is what we want. (this text describes just one way to do that)

Reply:
which is why the text below says "such as"

Reply:
@mcmanus: Please choose wording you like, put it in both places, and accept the PR.